This file is part of MXE.
See index.html for further information.

Submitted to upstream at
https://mailman.videolan.org/pipermail/libdvbpsi-devel/2014-June/000737.html

From 278634b51875f3b85817315bed05c448f852a70f Mon Sep 17 00:00:00 2001
From: Guilherme Lima Bernal <lb-guilherme@live.com>
Date: Tue, 24 Jun 2014 14:51:11 -0700
Subject: [PATCH 1/3] src/descriptor: fix include on MinGW

Signed-off-by: Timothy Gu <timothygu99@gmail.com>
---
 src/descriptor.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/descriptor.h b/src/descriptor.h
index f344a73..1d1364a 100644
--- a/src/descriptor.h
+++ b/src/descriptor.h
@@ -39,6 +39,8 @@
 extern "C" {
 #endif
 
+#include <sys/types.h>
+
 /*****************************************************************************
  * common definitions
  *****************************************************************************/
-- 
1.9.1

